common/src/GtinHandler.php

```diff
@@ -27,36 +20,70 @@ public function setProduct(\WC_Product $product) {
     }
 
     public static function getActivePlugin() {
-        foreach (self::SUPPORTED_PLUGINS as $plugin_name => $key) {
+        foreach ($this->getSupportedPlugins() as $plugin_name => $callback) {
```

Comment:
This will not work because the function is static. 

I will remove the `gtin_meta_key` from the constructor and make this function non-static.

Reply:
For example 5cbe13b